src/modules/launcher/Wox.Test/ResultsViewModelTest.cs

```diff
@@ -0,0 +1,170 @@
+﻿using NUnit.Framework;
+using PowerLauncher.ViewModel;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Wox.Core.Plugin;
+using Wox.Plugin;
+
+namespace Wox.Test
+{
+    [TestFixture]
+    class ResultsViewModelTest
+    {
+        [Test]
+        public void ResultsViewModel_HasNoContextMenuSelected_WhenInitialized()
```

Comment:
nit: WRT the naming convention.  I prefer to have the method being exectued in the name of the test.  The module identified in the filename.  A suggested edit would be ContextMenuSelectedIndex_ShouldEqualNoSelectionIndex_WhenInitialized.